### PR TITLE
[RFC] Disable animations in map to improve performance

### PR DESCRIPTION
**Pull request by dtinth**
Animating SVG color is costly and resulted in a sluggish animation on my computer...

**Before**
image

**After**
image

### CODE REVIEW

File: src/components/ElectionMap.js [hunk 1]
```diff
@@ -209,10 +201,7 @@ class ElectionMap extends SvgChart {
   }
 
   resetZoom() {
-    this.layers
-      .get("center/zoom")
-      .transition()
-      .attr("transform", "translate(0,0)scale(1)")
+    this.layers.get("center/zoom")
```
Comment (kristw) [inline src/components/ElectionMap.js:204]: This one ended up removing transform as well.

File: src/components/ElectionMap.js [hunk 2]
```diff
@@ -253,13 +241,11 @@ class ElectionMap extends SvgChart {
       .attr("y", -rectSide / 2)
       .attr("width", rectSide)
       .attr("height", rectSide)
-      .style("transition", "all .2s ease-out")
```
Comment (kristw) [inline src/components/ElectionMap.js:256]: this is only css

**Comment by kristw** [comment]
I will merge and fix the issues I marked.

File: src/components/ElectionMap.js (hunk 2 above)
Comment (dtinth) [reply src/components/ElectionMap.js:256]: Based on profile above, the most severe bottleneck is browser painting, not stuck on JavaScript execution krub.

**Pull request merged**